PENDING_RELEASE_NOTES.md

```diff
@@ -0,0 +1,10 @@
+
+## Version 14.10.0
+_Date TBD_
+
+### Data Modifications
+* CQM Criterion table
+  * Add new versions to 70 existing CQMs
+  * Add 2 new CQMs
+
+---
```

Comment:
Should be in PENDING_PENDING_RELEASE_NOTES, as STG is frozen. Also needs a new line at the end of the file